packages/card/src/react/index.tsx

```diff
@@ -0,0 +1,359 @@
+
+const Text: React.FC<React.HTMLAttributes<HTMLSpanElement>> = props => (
+  <span {...props} />
+)
+Text.displayName = 'Card.Text'
+
+const TextLink: React.FC<React.HTMLAttributes<HTMLSpanElement>> = props => {
+  const themeName = useTheme()
+  return <span {...styles.textLink(themeName)} {...props} />
+}
+TextLink.displayName = 'Card.TextLink'
+
+const Title: React.FC<React.HTMLAttributes<HTMLDivElement>> = props => {
+  const themeName = useTheme()
+  const { children, ...rest } = props
+
+  return (
+    <div {...styles.title(themeName)} {...rest}>
+      <Shiitake lines={2}>{children}</Shiitake>
+    </div>
+  )
+}
+Title.displayName = 'Card.Title'
+
+interface MetaDataProps {
+  metadata: (string | React.ReactElement<typeof TextLink>)[]
+  size: ValueOf<typeof vars.sizes>
+}
+const MetaData: React.FC<MetaDataProps> = props => {
+  const { metadata, size, ...rest } = props
+  const themeName = useTheme()
+  return (
+    <div {...styles.metadata(props, themeName)} {...rest}>
+      {metadata.map((m, i) => [
+        <span key={`datum${i}`} {...styles.metadataDatum()}>
+          {m}
+        </span>,
+        i < metadata.length - 1 && (
+          <span aria-hidden key={`dot${i}`} {...styles.metadataDot()}>
+            ·
+          </span>
+        )
+      ])}
+    </div>
+  )
+}
+
+export interface CardProps extends Record<string, unknown> {
+  actionBar?: React.ReactElement<typeof ActionBarAction>[]
+  actionBarVisible?: boolean
+  bonusBar?: React.ReactNode
+  fullOverlay?: React.ReactElement<typeof FullOverlayLink>
+  fullOverlayVisible?: boolean
+  image: React.ReactElement<typeof ImageLink>
+  metadata1?: (string | React.ReactElement<typeof TextLink>)[]
+  metadata2?: (string | React.ReactElement<typeof TextLink>)[]
+  progress?: number
+  size?: ValueOf<typeof vars.sizes>
+  tag?: React.ReactElement<typeof Tag>
+  title: React.ReactElement<typeof TextLink> | React.ReactElement<typeof Title>
+}
+
+interface CardStatics {
+  Action: typeof ActionBarAction
+  FullOverlayLink: typeof FullOverlayLink
+  Image: typeof Image
+  ImageLink: typeof ImageLink
+  sizes: typeof vars.sizes
+  Tag: typeof Tag
+  Text: typeof Text
+  TextLink: typeof TextLink
+  Title: typeof Title
+}
+const Card: React.FC<CardProps> & CardStatics = props => {
+  const htmlProps = omit<
+    CardProps,
+    [
+      'actionBar',
+      'actionBarVisible',
+      'bonusBar',
+      'fullOverlay',
+      'fullOverlayVisible',
+      'image',
+      'metadata1',
+      'metadata2',
+      'progress',
+      'size',
+      'tag',
+      'title'
+    ]
+  >(props, [
+    'actionBar',
+    'actionBarVisible',
+    'bonusBar',
+    'fullOverlay',
+    'fullOverlayVisible',
+    'image',
+    'metadata1',
+    'metadata2',
+    'progress',
+    'size',
+    'tag',
+    'title'
+  ])
+  const size = props.size || sizes.medium
+  return (
+    <div {...styles.card()} {...htmlProps}>
+      <div {...styles.overlays({ size })}>
+        {props.image ? props.image : null}
```

Comment:
is the null in the ternary because of strict mode...or could you do

`{props.image && props.image}`

or even just

`{props.image}`

Reply:
for tsc, yes. I can't remember if it was strict or unstrict that exposed this.